packages/web-react/src/components/Select/README.md

```diff
@@ -0,0 +1,69 @@
+# Select
+
+```jsx
+<Select id="example" name="example" validationState="danger" message="validation failed" isRequired>
```

Comment:
I suggest using a new prop instead of a deprecated one.

```suggestion
<Select id="example" name="example" validationState="danger" validationText="validation failed" isRequired>
```

Reply:
fixed